hazelcast/src/main/java/com/hazelcast/core/IAtomicLong.java

```diff
@@ -40,6 +40,15 @@
  * </pre>
  * </p>
  *
+ * During a network partition event it is possible for the register to exist in each of the partitioned clusters or to
```

Comment:
Should be done for the IAtomicReference as well.

Reply:
Yes, this will be coming in another pull request.  We're doing them bit by bit.